doc/userdoc/guides/recording_from_simulations.rst

```diff
@@ -35,87 +24,35 @@ temporal aspects of the simulation loop.
-using the additional properties ``start`` and ``stop``. These define
-activation and inactivation times of the device in ms. An additional
-property ``origin`` is available to shift the recording window by a
-certain amount, which can be useful in simulation protocols with
-repeated simulations. The following rules apply:
-
-- Collectors collect all events with timestamps `T` that fulfill
-
-  ::
-
-     start < T <= stop.
-
-  Events with timestamp `T == start` are not recorded.
-
-
-- Sampling devices sample at times *t = nh* (*h* being the simulation
-  resolution) with
-
-  ::
-
-     start < t <= stop
-     (t-start) mod interval == 0
-
-
-Common recorder properties
---------------------------
-
-All recorders have a set of common properties that can be set using
-``SetDefaults`` on the model class or ``SetStatus`` on a device
-instance:
-
-.. glossary::
-
- label
-   A string (default: `""`) specifying an arbitrary textual label for
-   the device.  Recording backends might use the label to generate
-   device specific identifiers like filenames and such.
-
- n_events
-   The number of events that were collected by the recorder can be
-   read out of the `n_events` entry. The number of events can be reset
-   to 0. Other values cannot be set.
-
- origin
-   A positive floating point number (default : `0.0`) used as the
-   reference time for `start` and `stop`.
+Recording devices can fundamentally be subdivided into two groups:
 
- record_to
-   A string (default: `"memory"`) containing the name of the recording
-   backend where to write data to. An empty string turns all recording
-   of individual events off.
+- **Collectors** gather events sent to them. Neurons are connected to
+  collectors and the collector gathers the events emitted by the
+  neurons connected to it.
 
- start
-   A positive floating point number (default: `0.0`) specifying the
-   activation time in ms, relative to `origin`.
+- **Samplers** actively interrogate their targets at given time
+  intervals. This means that the sampler must be connected to the
+  neuron (not the neuron to the sampler), and that the neuron must
+  support the particular type of sampling.
 
- stop
-   A floating point number (default: `infinity`) specifying the
-   deactication time in ms, relative to `origin`. The value of `stop`
-   must be greater than or equal to `start`
 
 Recorders for every-day situations
 ----------------------------------
 
-.. include:: ../models/multimeter.rst
-
-.. include:: ../models/spike_recorder.rst
-
-.. include:: ../models/weight_recorder.rst
+- :ref:`multimeter`
+- :ref:`spike_recorder`
+- :ref:`weight_recorder`
 
 .. _recording_backends:
 
 Where does data end up?
 -----------------------
 
-After a recording device has sampled or collected data, the *recording backends* are
-responsible for how the data are processed.
-
+After a recording device has sampled or collected data, the data is
```

Comment:
Minor point: Maybe `collected and sampled data` is better than `sampled and collected data` to preserve the order in which the groups of recording devices are introduced.